### DISCUSSION TITLE
Handling Sensitive Data and Security Best Practices

### DISCUSSION THREAD

**Discussion by Dorothy-Reyes** [pybamm-team/PyBaMM#3347] [General]
Hi,
Our [custom packaging site ](https://custompackaginglosangeles.com/)involves handling sensitive client data and I'm concerned about GitHub's security measures. How can I ensure robust data protection and implement best practices when working with confidential information on GitHub?

**Comment by brosaplanella** [comment]
Please see https://github.com/pybamm-team/PyBaMM/discussions/3353#discussioncomment-7032538
